webpack.config.test.js

```diff
@@ -22,5 +22,5 @@ module.exports = {
     }
   }),
   module: prodCfg.module,
-  devtool: "#source-map"
+  devtool: "source-map"
```

Comment:
These are equivalent, but `#` is the default so doesn't need to be specified. This matches our standard boilerplate.